src/store/modules/job/mutations.ts

```diff
@@ -44,6 +44,10 @@ const mutations: MutationTree <JobState> = {
     [types.JOB_MISCELLANEOUS_UPDATED] (state, payload){
         state.miscellaneous.list = payload.jobs;
         state.miscellaneous.total = payload.total;
+    },
+    [types.JOB_PIPELINE_FILTERS_UPDATED] (state, payload){
+        state.pipelineFilters.category = payload.category;
```

Comment:
I think we should just update pipelineFilters in mutation. It should be responsibility of actions to update specific filters. It will help when we introduce new filters

Reply:
Made the changes, sir. Also added refresh support on page change.